docs/api/03_plans/create-plan.mdx

```diff
@@ -45,40 +45,60 @@ curl --location --request POST "$LAGO_URL/api/v1/plans" \
           }
         },
         {
-        "billable_metric_id": "__BILLABLE_METRIC_ID__",
-        "charge_model": "graduated",
-        "properties": [
-          {
-            "to_value": 10,
-            "from_value": 0,
-            "flat_amount": "0",
-            "per_unit_amount": "0.00010"
-          },
-          {
-            "to_value": null,
-            "from_value": 11,
-            "flat_amount": "0",
-            "per_unit_amount": "0.0005"
+          "billable_metric_id": "__BILLABLE_METRIC_ID__",
+          "charge_model": "graduated",
+          "properties": [
+            {
+              "to_value": 10,
+              "from_value": 0,
+              "flat_amount": "0",
+              "per_unit_amount": "0.00010"
+            },
+            {
+              "to_value": null,
+              "from_value": 11,
+              "flat_amount": "0",
+              "per_unit_amount": "0.0005"
```

Comment:
I guess you meant "0.00005" otherwise the first tier is cheaper than the second one. (20 occurrences in the PR)